addon/lib/connect.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+import reduxStore from '../services/redux-store';
+
+const {
+  computed,
+  defineProperty,
+  inject: { service }
+} = Ember;
+
+export default function connect(mapStateToComputed, mapDispatchToActions, storeService) {
```

Comment:
Seems like each of these args is optional. Would a function signature like this be appropriate?
```javascript
function connection({ computedPropsMap, dispatchActionsMap, storeService }) {}
```
I'm thinking that it might be worthwhile to avoid requiring an empty `{}` to be passed in for `mapDispatchToActions` if what you need is `mapStateToComputed` and `storeService`, or empty `{}` for `mapStateToComputed` if all you need is `mapDispatchToActions`.

When I first read the function signature, `mapStateToComputed` felt like it was going to be a function because `map` originally struck me as a verb rather than a noun, so suggest renaming so that it's clear at a glance that this is a map object, not a map function.